dist/DynamicDataTable.js

```diff
@@ -105,10 +109,26 @@ class DynamicDataTable extends Component {
       }
     }
 
+    const regExpsToExclude = fieldsToExclude.filter(field => field.constructor && field.constructor === regExpsToExclude);
```

Comment:
```suggestion
    const regExsToExclude = fieldsToExclude.filter(field => field.constructor && field.constructor === regExsToExclude);
```

I'd prefer this pluralisation of `RegEx`.

Reply:
Never mind about this.

Reply:
JavaScript's constructor is `RegExp`